_includes/schema_org.html

```diff
@@ -0,0 +1,39 @@
+    <!-- Schema.org annotations in JSON-LD for a CreativeWork type.
+         Variables should be declared in the .md file 
+         For full list of available attributes see https://schema.org/CreativeWork -->
+    <script type="application/ld+json">
+    {
+       "@context": "http://schema.org/",
+       "@type": "creativeWork",
+       "genre": "trainingMaterial",
+       "name": "{{site.data.lesson.title}}",
+       "description": "{{site.data.lesson.description}}",
+       "keywords": "{{site.data.lesson.keywords}}",
+       "audience": [
+          {% for audience in site.data.lesson.audience %}{
+           "@type": "Audience",
+           "name": "{{audience}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
+       ],
+       "license": "{{site.data.lesson.licence}}",
+       "author": [
+         {% for author in site.data.lesson.author %}{
+           "@type": "Person",
+           "name": "{{author}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
+      ],
+       "contributor": [
+         {% for contributor in site.data.lesson.contributor %}{
+           "@type": "Person",
+           "name": "{{contributor}}"
+          }{% if forloop.last %}{%else%},{%endif%}
+          {% endfor %}
+      ],
+       "timeRequired": "{{site.data.lesson.timeRequired}}",
+       "learningResourceType": "{{site.data.lesson.learningResourceType}}",
+       "citation": "{{site.data.lesson.citation}}",
+       "dateCreated" : "{{site.data.lesson.dateCreated}}"
```

Comment:
It's not super clear what `dateCreated`  refers to. First commit? Last commit? Last release?

Reply:
That would be first commit after forking the lesson example.

Reply:
Why not use the date from the last commit? See ` [40f1dfa](/carpentries/lesson-example/pull/226/commits/40f1dfa82aa6b78c276543229a4db7b7239da142)`

Reply:
Last commit is not when the lesson has been created, it's when the lesson has been updated.
Why removing this field in 
https://github.com/carpentries/lesson-example/pull/226/commits/40f1dfa82aa6b78c276543229a4db7b7239da142

Reply:
@njall Does schema.org have a field for date updated? If not, can we use `dateCreated` for it?

Reply:
Yes, we have a field called dateModified that would be good for last commit. 

Date Created could be the first commit after the repo was cloned.

Reply:
OK. We can't automate `dateCreated`.

Reply:
I think that's ok, since it will happen once in a lifetime of the training meterial.